src/argilla_sdk/records/_dataset_records.py

```diff
@@ -338,16 +334,29 @@ def from_json(self, path: Union[Path, str]) -> "DatasetRecords":
             DatasetRecords: The DatasetRecords object created from the disk path.
 
         """
-        with open(path, "r") as f:
-            records = json.load(f)
-        self.update(records=[_dict_to_record(record) for record in records])
+        records = self()._records_from_json(path=path)
+        self.update(records)
         return self
 
+    def to_datasets(self) -> HFDataset:
```

Comment:
Since we need to add the same methods `to_datasets` `from_json` `to_json` mixins are not helping so much.